frameworks/cockroachdb/universe/config.json

```diff
@@ -0,0 +1,178 @@
+{
+  "type": "object",
+    "properties": {
+      "service": {
+        "type": "object",
+        "description": "DC/OS service configuration properties",
+        "properties": {
+          "name": {
+            "title": "Service name",
+            "description": "The name of the service instance",
+            "type": "string",
+            "default": "cockroachdb"
+          },
+          "user": {
+            "title": "User",
+            "description": "The user that the service will run as.",
+            "type": "string",
+            "default": "root"
+          },
+          "principal": {
+            "title": "Custom principal (optional)",
+            "description": "The principal for the service instance, or empty to use the default.",
+            "type": "string",
+            "default": ""
+          },
+          "secret_name": {
+            "title": "Credential secret name (optional)",
+            "description": "Name of the Secret Store credentials to use for DC/OS service authentication. This should be left empty unless service authentication is needed.",
+            "type": "string",
+            "default": ""
+          },
+          "version":{
+            "title": "CockroachDB version",
+            "description": "Production release version of CockroachDB",
+            "type": "string",
+            "default": "1.0.1"
```

Comment:
done. (not pushing to this branch though, since we are moving the work out of dcos-commons)